src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -578,6 +573,24 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
         }
         return responseFrequency;
     }
+    
+    private float[][] initializePercentageFrequenciesAndAverageValue(FeedbackRubricQuestionDetails fqd,
+                                    int[][] responseFrequency) {
```

Comment:
Also see if you can swap the parameters. It would be easier to follow if the parameter order is the same as the caller method.